### PR TITLE
Bug 1198246 - Avoid siteIcon resize r=gmarty

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=536188ffd8d7af02ac1f6629c969380f40530fbf

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8765187985c061cb3a5d7c27fc4cda82553e25e8

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f0428d222dce89ceef9e2a40cf82cb3f97d0a760

File: apps/system/style/chrome/images/loading_icon_white.svg [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+			c0.2,2.2,0.9,4.3,2,6.2l1.6-0.9C3.8,22.3,3.2,20.5,3,18.6z M9.2,5.2l-1-1.6c-2,1.3-3.7,3-4.9,5.1L5,9.6C6.1,7.8,7.5,6.3,9.2,5.2z
+			 M29.1,9.7l1.6-1c-1.1-1.9-2.7-3.5-4.4-4.8l-1,1.7C26.8,6.7,28.1,8.1,29.1,9.7z M4.1,11.2l-1.6-0.9c-0.9,2-1.4,4.1-1.5,6.4h1.9
+			C2.9,14.7,3.4,12.9,4.1,11.2z M24.9,28.7l0.9,1.6c1.6-1.1,3.1-2.5,4.2-4.1l-1.6-0.9C27.5,26.6,26.3,27.8,24.9,28.7z M17.9,2.9
+			C20,3,22,3.6,23.7,4.6l0.9-1.6c-2-1.1-4.3-1.8-6.7-1.9V2.9z M29.4,23.7l1.6,0.9c1-1.8,1.6-3.9,1.9-6.1H31
+			C30.8,20.4,30.3,22.1,29.4,23.7z M10.9,4.3c1.6-0.8,3.3-1.3,5.2-1.4V1c-2.2,0.1-4.3,0.7-6.2,1.6L10.9,4.3z M10.4,31.6
+			c1.8,0.8,3.7,1.3,5.7,1.4v-1.9c-1.7-0.1-3.3-0.5-4.8-1.2L10.4,31.6z M17.9,31.1V33c2.3-0.1,4.4-0.7,6.3-1.7l-0.9-1.6
+			C21.7,30.5,19.9,31,17.9,31.1z"/>
+	</g>
+</g>
+</svg>
```
Comment (gmarty) [inline apps/system/style/chrome/images/loading_icon_white.svg]: Can pass both icons to `svgo`? Just install it via `npm install svgo -g`, then optimise the folder doing `svgo -f apps/system/style/chrome/images/` from Gaia root folder.
Thanks!

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a269a96bf88fab56f553f5abb78c4155a089b362

File: apps/system/style/chrome/images/loading_icon_white.svg (hunk 1 above)
Comment (albertopq) [inline apps/system/style/chrome/images/loading_icon_white.svg]: Wow, great tip. Thanks!

**Pull request merged**